source/modules/.scaffold/scaffold.js

```diff
@@ -1,59 +1,60 @@
-/*!
+import EstaticoModule from '../../assets/js/helpers/module';
+
+const name = '{{name}}';
```

Comment:
I think we can skip this since the super class already adds a `name` property: https://github.com/unic/estatico/blob/develop/source/assets/js/helpers/module.js#L15